lib/shopify_cli.rb

```diff
@@ -48,6 +48,7 @@ module ShopifyCli
   TOOL_CONFIG_PATH = File.join(CONFIG_HOME, TOOL_NAME)
   LOG_FILE         = File.join(TOOL_CONFIG_PATH, 'logs', 'log.log')
   DEBUG_LOG_FILE   = File.join(TOOL_CONFIG_PATH, 'logs', 'debug.log')
+  DEFAULT_APP_URL  = 'https://shopify.github.io/shopify-app-cli/help/start-app/'
```

Comment:
This constant should not be part of the root namespace. This constant only has anything to do with the CreateApiClient task and if we never pass in the URL then we should just set this in that task and remove the parameter.